leaflet/index.d.ts

```diff
@@ -406,7 +406,7 @@ declare namespace L {
         getPane(name?: string): HTMLElement;
 
         // Popup methods
-        bindPopup(content: (layer: Layer) => Content | Content | Popup, options?: PopupOptions): this;
+        bindPopup(content: string | HTMLElement | Popup | ((layer: Layer) => Content), options?: PopupOptions): this;
```

Comment:
`Content` is an alias for `string | HTMLElement`, the previous definition just requires an extra set of parentheses around the function: `((layer: Layer) => Content) | Content | Popup`